addon/templates/components/feature-controls.hbs

```diff
@@ -11,7 +11,14 @@
       <tr>
         <td>{{flag.key}}</td>
         <td>{{flag.description}}</td>
-        <td>{{input type='checkbox' checked=flag.isEnabled change=(action 'doToggleFeature' flag.key flag.isEnabled) data-test-checkbox-flag=flag.key}}</td>
+        <td>
+          <label data-test-label-flag={{flag.key}}>
+            {{input type='checkbox' checked=flag.isEnabled change=(action 'doToggleFeature' flag.key flag.isEnabled) data-test-checkbox-flag=flag.key}}
+            {{#if (not-eq flag.isEnabled flag.default)}}
+              ❗
```

Comment:
Would you like to add some explanation? `<span title="This flag was overriden">❗</span>`

Reply:
:+1:

Reply:
+1